src/utils/useWeb3Connect.js

```diff
@@ -170,9 +174,15 @@ function useWeb3Connect() {
     const daiContract = new web3Inited.eth.Contract(daiAbi.abi, DAI_ADDRESS);
     setDaiContract(daiContract);
 
+    // console.log('daiContract');
```

Comment:
I think we can delete this now that we use the updateBalance function below.

Reply:
sure :)